sdk/Piwik_PRO_SDK_for_iOS.md

```diff
@@ -213,70 +213,70 @@ Ecommerce transactions (in-app purchases) can be tracked to help you improve you
 ```
 * An identifier (required) – a unique string identifying the order
 
-* A grandTotal (required) –  Total amount of the order, in cents
+* A grand total (required) –  The total amount of the order, in cents
 
-* A subTotal (optional) –  the subTotal (net price) for the order, in cents
+* A subtotal (optional) –  the subtotal (net price) for the order, in cents
 
-* A tax (optional) –  the tax for the order, in cents
+* Tax (optional) –  the tax for the order, in cents
 
-* A shipping (optional) –  the shipping for the order, in cents
+* Shipping (optional) –  the shipping for the order, in cents
 
-* A discount (optional) –  the discount for the order, in cents
+* Discount (optional) –  the discount for the order, in cents
 
-* Items (optional) –  the items included in the order, use ``addItemWithSku`` method to instantiate items
+* Items (optional) –  the items included in the order, use the ``addItemWithSku`` method to instantiate items
```

Comment:
Items in the example are listed in unexpected format (not attached to builder as items, but added as one element arrays). Can we confirm this is correct format?